build-system/status-page/comment.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Add progress comment for Stable and LTS cherry-picks
+ */
+
+const fetch = require('node-fetch');
+const [number, body, user] = process.argv.slice(2);
+const {getChannels, steps} = require('./common');
+const {log} = require('../common/logging');
+
+const commentTemplate = (channels) => `
+  #### 🌸 Cherry-Pick Progress 🌸 
+  Hi @${user}, thanks for filing this cherry-pick request!
+  Seeing that this affects ${channels.join(
+    ' and '
+  )}, [status.amp.dev](https://status.amp.dev) will be updated with progress of the fix.
+  Please update this tracker as each step is completed.
+  - [ ] ${steps[0].text}
+  - [ ] ${steps[1].text}
+  - [ ] ${steps[2].text}
+  - [ ] ${steps[3].text}
+  `;
+
+/**
+ * Add progress comment for Stable and LTS cherry-picks
+ * @return {Promise<object>}
```

Comment:
Maybe document the fields in this object via a typedef so the code can be type-checked?

Reply:
On re-looking at this function, should this annotation just be `Promise<void>`?